go/build.sh

```diff
@@ -2,6 +2,7 @@
 
 # may have to set GO111MODULE=off
 
+# export GO111MODULE=off
```

Comment:
We will need to turn it on to use go modules.  In next PRs we can  change this.

Reply:
ok